test/api/controllers/ControllerBaseSpec.scala

```diff
@@ -16,30 +16,31 @@
 
 package api.controllers
 
-import api.mocks.MockIdGenerator
 import api.models.audit.{AuditError, AuditEvent, AuditResponse}
+import api.models.errors.MtdError
 import api.models.domain.Nino
-import api.models.errors.{BadRequestError, ErrorWrapper, MtdError}
 import api.services.{MockAuditService, MockEnrolmentsAuthService, MockMtdIdLookupService}
-import cats.implicits.catsSyntaxValidatedId
+import config.RealAppConfig
+import config.MockAppConfig
 import config.Deprecation.NotDeprecated
-import mocks.MockAppConfig
 import play.api.http.{HeaderNames, MimeTypes, Status}
 import play.api.libs.json.{JsValue, Json}
 import play.api.mvc.{AnyContentAsEmpty, ControllerComponents, Result}
 import play.api.test.Helpers.stubControllerComponents
 import play.api.test.{FakeRequest, ResultExtractors}
-import routing.{Version, Version1}
 import support.UnitSpec
 import uk.gov.hmrc.http.HeaderCarrier
+import utils.MockIdGenerator
+import routing.{Version, Version1}
+import cats.implicits.catsSyntaxValidatedId
 
 import scala.concurrent.Future
 
-class ControllerBaseSpec extends UnitSpec with Status with MimeTypes with HeaderNames with ResultExtractors with MockAuditService with MockAppConfig {
+abstract class ControllerBaseSpec extends UnitSpec with Status with MimeTypes with HeaderNames with ResultExtractors with MockAuditService {
```

Comment:
Can ControllerBaseSpec be reformatted with scalafmt?

Reply:
everything is reformatted with scalafm. Do you think we need to format it differently?